src/LoginWithAuth0.ts

```diff
@@ -0,0 +1,82 @@
+import { Auth0Client } from "@auth0/auth0-spa-js"
+
+// Mapping of stages to Auth0 configurations
+const auth0ConfigMap = {
+  dev: {
+    domain: "https://auth.portal.dev.wandelbots.io",
+    clientId: process.env.AUTH0_DEV_CLIENT_ID,
+  },
+  stg: {
+    domain: "https://auth.portal.stg.wandelbots.io",
+    clientId: process.env.AUTH0_STG_CLIENT_ID,
+  },
+  prod: {
+    domain: "https://auth.portal.dev.wandelbots.io",
+    clientId: process.env.AUTH0_PROD_CLIENT_ID,
```

Comment:
Can we prefix this with WB?

Reply:
the other env variables are prefixed with another name :)